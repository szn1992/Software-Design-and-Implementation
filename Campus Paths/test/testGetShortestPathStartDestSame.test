# Test getting shortest path when abbreviated names of start
# and dest are the same
r
CSE
CSE
q