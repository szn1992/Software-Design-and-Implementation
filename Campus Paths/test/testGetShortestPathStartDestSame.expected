Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu): # Test getting shortest path when abbreviated names of start
# and dest are the same
Abbreviated name of starting building: Abbreviated name of ending building: Path from Paul G. Allen Center for Computer Science & Engineering to Paul G. Allen Center for Computer Science & Engineering:
Total distance: 0 feet

Enter an option ('m' to see the menu): 